bricks/dart_frog_prod_server/hooks/test/post_gen_test.dart

```diff
@@ -30,13 +30,19 @@ class _MockLogger extends Mock implements Logger {}
 
 class _MockProgress extends Mock implements Progress {}
 
-class _MockProcessResult extends Mock implements ProcessResult {}
-
 void main() {
   group('postGen', () {
     late HookContext context;
     late Logger logger;
 
+    const processId = 42;
```

Comment:
Is this 42 an arbitrary number?

Reply:
https://medium.com/@lgm527/42-cbcb2773bcab 😄

Reply:
SGTM as long as you have a towel!